addons/account/models/account_account.py

```diff
@@ -558,7 +599,7 @@ def _set_opening_balance(self):
             account._set_opening_debit_credit(abs(balance) if balance > 0.0 else 0.0, 'debit')
             account._set_opening_debit_credit(abs(balance) if balance < 0.0 else 0.0, 'credit')
```

Comment:
You don't use the new param `company`?
I don't think it is possible to make it work though